src/Nancy.Swagger/Services/RouteUtils/SwaggerRouteDescriber.cs

```diff
@@ -77,6 +77,10 @@ public void AddAdditionalModels(params Type[] types)
                             {
                                 param.Type = "string";
                             }
+                            if(param.Type == "file")
+                            {
+                                op.ConsumeMimeType("multipart/form-data");
```

Comment:
Not sure if it matters but technically the file type can also consume application/x-www-form-urlencoded.

I think for all intents and purposes they're basically the same though